microcosm_logging/levels.py

```diff
@@ -0,0 +1,53 @@
+"""
+Logging levels
+
+"""
+from functools import total_ordering
+
+
+@total_ordering
+class ConditionalLoggingLevel(object):
+    """
+    A logging level that is conditional on a function.
+
+    Works by masquerading as an int.
```

Comment:
To be fair is it masquerading as an int or just implementing a total ordering so `lt` and `eq` work?